a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -286,34 +286,48 @@ <h5>Variables setup</h5>
                         </li>
                     	
                     	<li>
-                    		<b>LET</b> <var>has_nonvisual_metadata</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property=("<i>schema:accessMode</i>", "<i>schema:accessModeSufficient</i>")]</code>.
+                    		<b>LET</b> <var>audio_only_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space() = "auditory" and count(//meta[@property="schema:<i>accessMode</i>"]) = 1]</code>.
```

Comment:
or accessModeSufficient="auditory" right?

Reply:
No, this is to say that the book is not readable in read aloud or refreshable braille. If we look at sufficient access modes, we could end up catching text publications with media overlays, which we wouldn't want to flag as not readable.

That's not true of visual where you could have only a primary access mode of visual but there's alt text on the images so it has somewhat readable text.

Similarly, if you only check for a textual access mode you'd miss mixed text with visual/auditory publications where the text alternatives are sufficient. That's why we need to check both access mode and access mode sufficient to see if all the content is readable as text.